esp/esp/web/util/latex.py

```diff
@@ -149,7 +150,11 @@ def _gen_latex(texcode, stdout, stderr, type='pdf', remove_files=False):
 
     elif type == 'log':
         mime = 'text/plain'
-        call(['latex'] + latex_options + ['%s.tex' % file_base])
+        # Use `subprocess.call()`, not `subprocess.check_call()`, because the
+        # command doesn't need to succeed in order to read the generated .log
+        # file. In fact, for the use case of this type (trying to debug why
+        # compilation isn't working), it is likely that the command will fail.
+        subprocess.call(['latex'] + latex_options + ['%s.tex' % file_base], cwd=TEX_TEMP, stdout=stdout, stderr=stderr)
```

Comment:
It looks like it's just a `partial` defined above, so it might be better to just rename it to `check_call` and then make a similar `call` wrapper.